crates/type-system/.eslintrc.cjs

```diff
@@ -0,0 +1,4 @@
+/** @type {import("eslint").Linter.Config} */
+module.exports = {
+  ...require("@local/eslint-config/generate-workspace-config.cjs")(__dirname),
```

Comment:
Using spread operation instead of `module.exports = require(...)` for future-proofing. This way it is easier to add custom rules while keeping diff small. Besides, `require` fits one line so is not chunked by Prettier.